tests/phpt/error_handler_captures_exception_thrown_from_next_handler.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+
```

Comment:
Description missing

Reply:
Gonna delete this test as part of the cleanup of this PR from the changes we decided to drop